### PR TITLE
fix NumberFormat exception in MysqlURLParser

### PR DESCRIPTION
**Pull request by wuguanyu**
Fix NumberFormat in MysqlURLParser
https://github.com/apache/skywalking/issues/7758

### CODE REVIEW

**Comment by wu-sheng** [comment]
changes.md in the root should be updated too.

File: apm-sniffer/apm-sdk-plugin/jdbc-commons/src/main/java/org/apache/skywalking/apm/plugin/jdbc/connectionurl/parser/MysqlURLParser.java [hunk 1]
```diff
@@ -99,11 +110,13 @@ public ConnectionInfo parse() {
...
-                        .endIndex()));
+                return new ConnectionInfo(component, dbType, hostAndPort[0], Integer.valueOf(hostAndPort[1]),
+                                          fetchDatabaseNameFromURL(location
+                                                                           .endIndex()));
             } else {
-                return new ConnectionInfo(component, dbType, hostAndPort[0], DEFAULT_PORT, fetchDatabaseNameFromURL(location
-                        .endIndex()));
+                return new ConnectionInfo(component, dbType, hostAndPort[0], DEFAULT_PORT,
+                                          fetchDatabaseNameFromURL(location
+                                                                           .endIndex()));
```
Comment (wu-sheng) [inline apm-sniffer/apm-sdk-plugin/jdbc-commons/src/main/java/org/apache/skywalking/apm/plugin/jdbc/connectionurl/parser/MysqlURLParser.java:119]: Do you mis-format these lines? They seem no change. There is a style file in the root folder, please import them.

Comment (wuguanyu) [reply apm-sniffer/apm-sdk-plugin/jdbc-commons/src/main/java/org/apache/skywalking/apm/plugin/jdbc/connectionurl/parser/MysqlURLParser.java:119]: ok

**Review by wu-sheng — APPROVED** [review]
Forward @ascrutae 's approval, his account has an issue.

**Pull request merged**